test/e2e/page-objects/pages/settings/privacy-settings.ts

```diff
@@ -27,6 +27,19 @@ class PrivacySettings {
     tag: 'button',
   };
 
+  private readonly dataCollectionForMarketingToggle =
+    '[data-testid="data-collection-for-marketing-toggle"] .toggle-button';
+
+  private readonly dataCollectionWarningAckButton = {
+    text: 'Okay',
+    tag: 'Button',
```

Comment:
nit: it seems it's working regardless, so just cosmetics, in case we want to change it:

```suggestion
    tag: 'button',
```

Reply:
addressed, thanks !